instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -146,6 +146,7 @@ def response_hook(span, req, resp):
 from typing import Collection
 
 import falcon
+from packaging import version as package_version
```

Comment:
Is this part of all python distributions? If not, we'll need to add it as a dependency to setup.cfg

Reply:
Good catch https://github.com/open-telemetry/opentelemetry-python-contrib/issues/881